mysql-test/suite/mariabackup/mdev-14447.test

```diff
@@ -9,6 +9,7 @@ let $incremental_dir=$MYSQLTEST_VARDIR/tmp/backup_inc1;
 SET GLOBAL innodb_file_per_table=0;
 CREATE TABLE t(a varchar(40) PRIMARY KEY, b varchar(40), c varchar(40), d varchar(40), index(b,c,d)) ENGINE INNODB;
 
+sleep 1000;
 echo # Create full backup , modify table, then create incremental/differential backup;
 --disable_result_log
 exec $XTRABACKUP --defaults-file=$MYSQLTEST_VARDIR/my.cnf  --backup --parallel=10 --target-dir=$basedir;
```

Comment:
What is the purpose of this? Generally, relying on timing is a bad idea. At least there needs to be a comment about this, and preferrably we should poll for a speific event rather than wait some time, which might be too long in some environments and too short in others.

Reply:
sorry, I wasn't trying to repeat some issue. I wrongly pushed this patch.